### PR TITLE
v3.5.0-beta.1

### PR DESCRIPTION
**Pull request by RingerJK**
### Description
upstream v3.5.0-beta.1

df5e110e53346a83f45e84a26c8f38e42da99aed

### CODE REVIEW

**Review by DzmitryFomchyn — APPROVED** [review]

File: CODEOWNERS
Comment (Zayankovsky) [inline CODEOWNERS:1]: Why did you remove CODEOWNERS?

Comment (Zayankovsky) [inline CODEOWNERS:1]: @RingerJK why did you remove CODEOWNERS?

Comment (RingerJK) [reply CODEOWNERS:1]: CODEOWNERS does not exist in the Monorepo. It should be added there, so upstream codebase contain it. I'll take it
